github/github-accessors.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The go-github AUTHORS. All rights reserved.
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
```

Comment:
Sorry, this was fixed in #825 but you would need to have sync'd first to get the fix (before running the generator).
Please manually change this back to 2017 and Travis CI should be happy.